transaction/std/src/memo/gift_code_funding.rs

```diff
@@ -136,70 +174,89 @@ mod tests {
 
         // Create memo with blank note
         let note = "";
-        let memo = GiftCodeFundingMemo::new(&key, note).unwrap();
-
-        // Check that the note is extracted properly
-        assert_eq!(memo.funding_note().unwrap(), note);
+        let fee = 666;
+        let memo = GiftCodeFundingMemo::new(&key, fee, note).unwrap();
 
         // Check that the public key can be verified
         assert!(memo.public_key_matches(&key));
+
+        // Check the fee is correct
+        assert_eq!(memo.get_fee(), 666);
+
+        // Check that the note is correct
+        assert_eq!(memo.funding_note().unwrap(), note);
     }
 
     #[test]
-    fn test_gift_code_funding_memo_with_only_null_memo_bytes_is_okay() {
+    fn test_gift_code_funding_memo_note_with_only_null_memo_bytes_is_okay() {
         // Initialize hash bytes
         let mut rng: StdRng = SeedableRng::from_seed([1u8; 32]);
         let key = RistrettoPublic::from_random(&mut rng);
+        let fee: u64 = 666;
         let hash_bytes = tx_out_public_key_short_hash(&key);
+        let fee_bytes = fee.to_be_bytes();
 
-        // Put only hash bytes into memo_bytes, leaving the rest empty & make memo
-        // object
+        // Put only hash and fee bytes into memo_bytes, leaving note bytes empty
         let mut memo_bytes = [0u8; GiftCodeFundingMemo::MEMO_DATA_LEN];
-        memo_bytes[0..GiftCodeFundingMemo::HASH_DATA_LEN].copy_from_slice(&hash_bytes);
-        let memo = GiftCodeFundingMemo::from(&memo_bytes);
+        memo_bytes[..GiftCodeFundingMemo::HASH_DATA_LEN].copy_from_slice(&hash_bytes);
+        memo_bytes[GiftCodeFundingMemo::HASH_DATA_LEN..GiftCodeFundingMemo::NOTE_OFFSET]
+            .copy_from_slice(&fee_bytes[1..]);
 
-        // Check that a blank note is extracted properly
-        let note = "";
-        assert_eq!(memo.funding_note().unwrap(), note);
+        let memo = GiftCodeFundingMemo::from(&memo_bytes);
 
         // Check that the public key can be verified
         assert!(memo.public_key_matches(&key));
+
+        // Check that the fee is correct
+        assert_eq!(memo.get_fee(), fee);
+
+        // Check that a blank note is extracted correctly
+        let note = "";
+        assert_eq!(memo.funding_note().unwrap(), note);
```

Comment:
Nit Inline
```suggestion
        assert_eq!(memo.funding_note().unwrap(), "");
```

Reply:
I prefered to use the variable to keep it consistent with the rest of the tests